Source/WebCore/CMakeLists.txt

```diff
@@ -2536,38 +2536,38 @@ add_custom_command(
     OUTPUT ${WebCore_DERIVED_SOURCES_DIR}/CSSValueKeywords.in ${WebCore_DERIVED_SOURCES_DIR}/CSSValueKeywords.h ${WebCore_DERIVED_SOURCES_DIR}/CSSValueKeywords.cpp ${WebCore_DERIVED_SOURCES_DIR}/CSSValueKeywords.gperf
     MAIN_DEPENDENCY ${WEBCORE_DIR}/css/process-css-values.py
     DEPENDS ${WebCore_CSS_VALUE_KEYWORDS}
+    DEPENDS ${WEBCORE_DIR}/css/CSSPrimitiveValueMappings.h
+    DEPENDS ${WEBCORE_DIR}/css/parser/CSSParser.cpp
     WORKING_DIRECTORY ${WebCore_DERIVED_SOURCES_DIR}
     COMMAND ${PERL_EXECUTABLE} -ne "print" ${WebCore_CSS_VALUE_KEYWORDS} > ${WebCore_DERIVED_SOURCES_DIR}/CSSValueKeywords.in
     COMMAND ${PYTHON_EXECUTABLE} ${WEBCORE_DIR}/css/process-css-values.py --defines "${FEATURE_DEFINES_WITH_SPACE_SEPARATOR} ${CSS_VALUE_PLATFORM_DEFINES}" --gperf-executable "${GPERF_EXECUTABLE}"
     VERBATIM)
 list(APPEND WebCore_SOURCES ${WebCore_DERIVED_SOURCES_DIR}/CSSValueKeywords.cpp)
-ADD_SOURCE_WEBCORE_DERIVED_DEPENDENCIES(${WEBCORE_DIR}/css/parser/CSSParser.cpp CSSValueKeywords.h)
-ADD_SOURCE_WEBCORE_DERIVED_DEPENDENCIES(${WEBCORE_DIR}/css/CSSPrimitiveValueMappings.h CSSValueKeywords.h)
 
 # Generate code and maps for CSS pseudo class & element selectors.
 add_custom_command(
     OUTPUT ${WebCore_DERIVED_SOURCES_DIR}/CSSSelectorEnums.h ${WebCore_DERIVED_SOURCES_DIR}/CSSSelectorInlines.h ${WebCore_DERIVED_SOURCES_DIR}/SelectorPseudoClassAndCompatibilityElementMap.gperf ${WebCore_DERIVED_SOURCES_DIR}/SelectorPseudoClassAndCompatibilityElementMap.cpp ${WebCore_DERIVED_SOURCES_DIR}/SelectorPseudoElementMap.gperf ${WebCore_DERIVED_SOURCES_DIR}/SelectorPseudoElementMap.cpp
     MAIN_DEPENDENCY ${WEBCORE_DIR}/css/CSSPseudoSelectors.json
+    DEPENDS ${WEBCORE_DIR}/css/CSSSelector.cpp
+    DEPENDS ${WEBCORE_DIR}/css/parser/CSSSelectorParser.cpp
```

Comment:
I think we can actually remove these based on what you said, since they don't affect CSSSelectorInlines.h or CSSSelectorEnums.h. The dependency is the other way round: CSSSelectorInlines.h/ CSSSelectorEnums.h affects those files.

Reply:
Well, drat.

Reply:
It looks like I mixed up source vs. deps everywhere throughout this pull request, which is unimpressive. Oops. I'm going to revert and try again.

Reply:
Follow up: https://bugs.webkit.org/show_bug.cgi?id=266998